### PR TITLE
fix(storefront): BCTHEME-1906 High severity security issue

### PR DESCRIPTION
**Pull request by bc-vlad-dlogush**
Jira: [BCTHEME-1906](https://bigcommercecloud.atlassian.net/browse/BCTHEME-1906)

## What/Why?
This PR updates the affected packages

[BCTHEME-1906]: https://bigcommercecloud.atlassian.net/browse/BCTHEME-1906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by BC-krasnoshapka — APPROVED** [review]

**Comment by BC-krasnoshapka** [comment]
@VladDlogush there is an error during build most likely related to this changes:

> Validation Error:
>  Test environment jest-environment-jsdom cannot be found. Make sure the testEnvironment configuration option points to an existing node module.

**Comment by sacr3dc0w** [comment]
> @VladDlogush there is an error during build most likely related to this changes:
> 
> > Validation Error:
> > Test environment jest-environment-jsdom cannot be found. Make sure the testEnvironment configuration option points to an existing node module.

I had this issue on another project. Jest no longer ships `jsdom`, or `jasmine2`. To resolve, run `npm i jest-environment-jsdom jest-jasmine2 --save-dev` to add to the dependency tree.

File: jest.setup.js [hunk 1]
```diff
@@ -18,6 +18,5 @@ window.Element.prototype.getClientRects = function () {
         }
         node = node.parentNode;
     }
-    var self = $(this);
-    return [{ width: self.width(), height: self.height() }];
+    return [{ width: this.offsetWidth, height: this.offsetHeight }];
 };
```
Comment (bc-yevhenii-buliuk) [inline jest.setup.js:22]: @bc-vlad-dlogush add please an extra line

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Draft
+- High severity security issue [#2476](https://github.com/bigcommerce/cornerstone/pull/2476)
```
Comment (bc-yevhenii-buliuk) [inline CHANGELOG.md:8]: @bc-vlad-dlogush please don't forget set appropriate PR's number

**Review by bc-yevhenii-buliuk — APPROVED** [review]

**Review by yurytut1993 — APPROVED** [review]

**Review by bc-azvierieva — APPROVED** [review]

**Comment by mattcoy-arcticleaf** [comment]
@bc-vlad-dlogush lighthouse 12 is not compatible with Node 18.15. Should this be rolled back or should the .nvmrc file be updated to a newer version of Node?

**Pull request merged**